.github/scripts/printPodspec.rb

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ruby
+
+# This file is a lightweight port of the `pod ipc spec` command.
```

Comment:
worth noting that the upstream `pod ipc spec` command hasn't changed in over 8 years: https://github.com/CocoaPods/CocoaPods/blob/028af0bdfc56df9e1b221a59cf36306690cf2ce4/lib/cocoapods/command/ipc/spec.rb

so this should hopefully be pretty stable code